install/infra/modules/gke/cluster.tf

```
@@ -149,3 +149,18 @@ resource "local_file" "kubeconfig" {
   filename = var.kubeconfig
   content  = module.gke_auth.kubeconfig_raw
 }
+
+resource "google_service_account" "cluster_user_sa" {
+  account_id   = local.gke_user_sa
+  display_name = "Service Account managed by TF for GKE cluster user"
```

```suggestion
  display_name = "Gitpod Service Account managed by TF for GKE cluster user"
```

Might be worth mentioning Gitpod as I _THINK_ that these appear in a bit long list. Could get confusing for people managing a list with lost of service accounts